src/commands/scratch-create.yml

```diff
@@ -0,0 +1,22 @@
+# Creates a scratch SFDX Org.
+description: >
+  'Create a scratch Salesforce Org.'
+parameters:
+  scratch-config:
+    description: 'The scratch JSON definition.'
+    type: string
+  scratch-alias:
+    description: 'The alias of the scratch Org.'
+    type: string
+  overrides:
+    description: 'Overrides for scratch organization.'
+    type: string
+    default: ''
+
+steps:
+  - run:
+      alias: Create << parameters.scratch-alias >> SalesForce Org
+      command: sfdx force:org:create -f << parameters.scratch-config >> -a << parameters.scratch-alias >> <<# parameters.overrides >> << parameters.overrides >> <</ parameters.overrides >>
+  - run:
+      alias: Ensure existance of org & print values.
```

Comment:
This should be combined into a single run-step. There is no need to create two separate steps in the UI for this from the user perspective.

Reply:
Very true - on it!